examples/minimal-winit/Cargo.toml

```diff
@@ -2,17 +2,16 @@
 name = "minimal-winit"
 version = "0.1.0"
 authors = ["Jay Oster <jay@kodewerx.org>"]
-edition = "2021"
+edition = "2024"
 publish = false
 
 [features]
 optimize = ["log/release_max_level_warn"]
 default = ["optimize"]
 
 [dependencies]
-env_logger = "0.10"
-error-iter = "0.4"
 log = "0.4"
+env_logger = "0.11"
+error-iter = "0.4"
```

Comment:
Changing the order of the dependencies is unnecessary. They were ordered alphabetically.